cellpack/autopack/Analysis.py

```diff
@@ -890,7 +900,7 @@ def PairCorrelationFunction_3D(self, data, S, rMax, dr):
         interior_y y coordinates of reference particles
         interior_z z coordinates of reference particles
         """
-        from numpy import zeros, sqrt, where, pi, average, arange, histogram
+        from numpy import arange, average, histogram, pi, sqrt, where, zeros
```

Comment:
[nit] - these imports should be at the top of the file

Reply:
Huh, looks like these were imported automatically. I'll check in this.

Reply:
Looks like these functions are leftover from the original codebase. Not sure if they're in use anymore. Might be worth going over them at some point to check their coverage.